README.md

```diff
@@ -34,9 +34,6 @@ The [TAG Contributor Strategy charter](/CHARTER.md) outlines the scope of our gr
 We have one general meeting a month, along with meetings for our working groups.
 Go to [the CNCF calendar](https://tockify.com/cncf.public.events/monthly?search=Contributor+Strategy) to see when we meet next.
 
-- Calendar invites are sent to the mailing list(see: [Communicating with us](#communicating-with-us)).
-Once you join, you won't automatically have the invite on your calendar. You can
-get it from a [past message here](https://lists.cncf.io/g/cncf-tag-contributor-strategy/message/1)
```

Comment:
I would be happy to undelete and update this part, if somebody knows how to get an invite for the up-to-date meeting.

I deleted the text all over the place, that talks about getting a calendar invite.

Reply:
+1.

I guess we're good then: I removed the link to the message, but kept the link to Tockify ("View our calendar to see when we meet next." text below).